T0/T0calib/AliT0TimeAmplCorr.cxx

```diff
@@ -400,7 +400,7 @@ Int_t AliT0TimeAmplCorr::SetNewQTCcoefficients(TString filename)
             if((runMax - runMin) < minRUNdiff) enteryChoosen = enteryForMeanValues;
             tree_coef->GetEntry(enteryChoosen);
 
-            for(Int_t pmt = 0; pmt <= NPMT0; pmt++)
```

Comment:
Are you sure? I mean: it makes sense to me, and I believe you addressed a specific compiler warning here, but we must be sure this really does what we want.

Reply:
Yes, I checked that, and actually it caused a buffer overflow before.